website/views/shared/tasks/edit/rewards/pricing.jade

```diff
@@ -1,4 +1,4 @@
-fieldset.option-group.option-short(ng-if='task.type=="reward" && !task._edit.challenge.id')
+fieldset.option-group.option-short(ng-if='task.type=="reward" && challenge_edit')
```

Comment:
Pretty sure this will prevent users from editing their rewards on their task page.